### ISSUE TITLE
Please add MUSICWHEELITEM Grades

### ISSUE THREAD

**Issue by dougshell** [dguzek/Simply-Love-SM5#81]
Hello and thank you for your beautiful theme.

tldr; Please add grading icons similar to the screenshot below only with ITG Grading scheme (S-/S-S+/star-quadstar) and in white (or a color of your choosing, user definable if possible.

I have no clue what I am doing here or if I am doing this wrong, but i will explain. In many themes, you can see grade based scores for all songs visible during the music selection screen for a given difficulty selection. (eg. If you select Hard all scores that have been passed on hard will show a visible score, if you switch to expert, all your grades for expert will appear).

I have attempted to pull the required files and LUA changes from Lambda the new default theme for 5.1, but this solution brings many challenges and regressions.

First, the images do not fit the beautiful styling of Simply Love (theme styling and ITG grading brackets). Further, while bringing the metrics.ini adjustments as well as required files from Lambda does in fact show grades, it does not allow your theme to create NEW grades, no does it allow Lambda to pick up grades that have been created in Simply Love.

This may be because there are many more grading brackets in Simply Love than there are in the default theme.

I do not expect you to add this theme feature just for me, but I was hoping for a bit of advice. Could i fix these issues be simply creating a grades.png file that is 2x17 vice 2x8 as in standard in the default theme? If that is the case i will learn to use Photoshop to import your assets into a 2x17 grid "grades 2x17.png" file. 

I apologize for the long issue request, but i wanted to explain as best as i can.

I have attached a screenshot of what I have figured out so far. Since that screenshot I have adjusted the Y value to properly align the grade images.

Thank you in advance for your consideration in this matter.

![2018-01-05_175918](https://user-images.githubusercontent.com/4993555/34644388-b2845cd2-f303-11e7-9d90-d3dea4bb43b3.jpg)

**Comment by dguzek** [comment]
> tldr; Please add grading icons similar to the screenshot below only with ITG Grading scheme (S-/S-S+/star-quadstar) and in white (or a color of your choosing, user definable if possible.

This is not a feature I am interested in adding to the theme at this time.

Simply Love is still _very_ rooted in ITG gameplay.  The primary user-base is almost entirely post-ITG players, and letter grades aren't nearly as meaningful in ITG as _percent scores_ are.  (Some enterprising users have added percent scores to Simply Love's MusicWheelItem code, but I'm unsure if any of these derivatives are actually on GitHub.)

Additionally, I do not have the time to guide you through a learn-to-theme exercise.  (I wish I did, as programming education is something I really enjoy.)

Making a `2x17` spritesheet of letter grade graphics might automatically work with your existing code; it might not.  You should try it and find out. 😊

I am closing this issue now.

**Comment by dougshell** [comment]
Thank you for your indepth response. Regardless of your desire to implement
this feature request, you have responded in a way that encourages me to
learn to at least make a novice attempt to implement this feature.

I have looked at the list documentation, and while I most admit I am
certainly overwhelmed, I will continue to learn. Once again, thank you
